roles/swift-ring/tasks/main.yml

```diff
@@ -13,3 +13,4 @@
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
            bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
   when: not should_boostrap|success
+  notify: restart swift services
```

Comment:
So a command task will always result in a change, unless a changed_when control is used. However, the current conditional here would only set up the rings once, so maybe that's okay.

In what scenario are you seeing that the creation of the rings requires the service to restart? Why would we only see this in source based installs and not package installs?